streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java

```diff
@@ -488,6 +488,30 @@ public void shouldCheckpointRestoredOffsetsToFile() throws IOException {
         assertThat(readOffsetsCheckpoint(), equalTo(checkpointMap));
     }
 
+    @Test
+    public void shouldSkipGlobalInMemoryStoreOffsetsToFile() throws IOException {
+        stateManager.initialize();
+        initializeConsumer(10, 1, t3);
+        stateManager.register(store3, stateRestoreCallback);
+        stateManager.close(Collections.<TopicPartition, Long>emptyMap());
+
+        final OffsetCheckpoint checkpoint = new OffsetCheckpoint(checkpointFile);
+
+        assertThat(checkpoint.read(), equalTo(Collections.<TopicPartition, Long>emptyMap()));
```

Comment:
Can this be simplified to `assertThat(readOffsetsCheckpoint(), equalTo(Collections.<TopicPartition, Long>emptyMap()));` and remove variable `final OffsetCheckpoint checkpoint = new OffsetCheckpoint(checkpointFile);` ?